src/java.base/share/classes/java/lang/StrictMath.java

```diff
@@ -42,8 +42,28 @@
  * Library," <a
  * href="https://www.netlib.org/fdlibm/">{@code fdlibm}</a>. These
  * algorithms, which are written in the C programming language, are
- * then to be understood as executed with all floating-point
- * operations following the rules of Java floating-point arithmetic.
+ * then to be understood to be transliterated into Java and executed
```

Comment:
This change looks good (did not build and view javadoc).

Reply:
I checked that the change rendered as desired in a local docs build.